packages/coinstac-ui/app/render/components/user-account-controller.js

```diff
@@ -38,7 +37,7 @@ class UserAccountController extends Component {
 
   render() {
     return (
-      <UserAccount logout={this.logout} {...this.props} />
```

Comment:
Adding a new prop called `logout` to take the place of the previously dispatched action `logout` meant that when we pass into `UserAccount` a function named `logout` referencing AccountController's local function in addition to using the spread operator to pass in all the props, including the new `logout` prop, we see issues. The UserAccount component would have two props named `logout`. So, I just renamed the local function.